cpp/simulations/abm.cpp

```diff
@@ -807,33 +709,35 @@ mio::IOResult<void> run(const fs::path& result_dir, size_t num_runs, bool save_s
     auto tmax             = mio::abm::TimePoint(0) + mio::abm::days(60); // End time per simulation
```

Comment:
One important thing I forgot yesterday: There is no structure like `parameter_studies.h` (see `memilio/compartments`) and some of this structures should maybe be implemented to simplify these ensemble run calls. Note that the https://github.com/DLR-SC/memilio/blob/main/cpp/memilio/compartments/parameter_studies.h already handles graph-ODE and MPI, both of it is maybe not necessary for the first ABM implementation.

Reply:
Yes, I agree. It should be in a separate issue since this one is already quite long.

Reply:
Yes. I now have a new issue, #759, for this. I will close it for now, but could you have a talk later with Martin and update it? Thanks!